manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "A self hostable read-it-later app",
         "fr": "Une application de lecture-plus-tard auto-hébergeable"
     },
-    "version": "2.3.7-1~ynh2",
+    "version": "2.3.8-1~ynh2",
```

Comment:
I think the version number should be "2.3.8~ynh1"
The upstream version is 2.3.8 and not 2.3.8-1 (or I miss something?)
And the "yuno version number": ~ynhX is reset each time the upstream version change.

Reply:
Oh, I changed the version number without even paying attention that we were in ~ynh2 ^^